etl-federation/README.md

```diff
@@ -2,9 +2,14 @@
 
 These are the steps to enable ETL federation with Kubecost.
 
-This is an efficient method to implement multi-cluster Kubecost while using existing prometheus installations. Contact us for help customizing settings.
+This is an efficient method to implement multi-cluster Kubecost while using existing prometheus installations. The example in this directory uses the Kubecost Prometheus server. See the [existing-prometheus](./existing-prometheus/) directory for additional configuration required.
```

Comment:
I would suggest to clarify it more about the purpose of Prometheus, so customers do not think that they need to manage another Prometheus instance. Example:
`Federated-ETL is an efficient method to implement multi-cluster Kubecost while using existing Prometheus installations for metrics collection only.`

Reply:
great suggestion!